toml/i_want_go_home.toml

```
@@ -0,0 +1,2 @@
+[i]
+want = { go = "home" }
```

TOML은 잘 모르지만...
```suggestion
i = { want = "go home" }
```
쪽이 더 의미가 어울리지 않나 싶습니다

---

음, 그렇네요.
굳이 namespace 같은 느낌으로 하는 것 보다는 더 나아 보이네요!